ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx

```diff
@@ -74,17 +74,7 @@ export const WorkflowNodeSummary = (props: Props) => {
         attributes.splice(2, 0, {title: 'POD NAME', value: props.node.id});
         attributes.push({
```

Comment:
Shouldn't this be surrounded with a `if props.node.resourcesDuration !== ""/nil`? I don't like how we do the check inside the `ResourcesDuration` object. If it empty, "RESOURCES DURATION" is still displayed, no?

Reply:
good question - this feature can be disabled - should we hide it if disabled - or provide a link to how to enable it?

Reply:
Hide if disabled is my vote

Reply:
this will be empty if disabled, so I'll guard on that

Reply:
done